cicecore/cicedynB/infrastructure/ice_memusage_gptl.c

```diff
@@ -0,0 +1,215 @@
+/*
+** $Id: get_memusage.c,v 1.10 2010-11-09 19:08:53 rosinski Exp $
+**
+** Author: Jim Rosinski
+**   Credit to Chuck Bardeen for MACOS section (__APPLE__ ifdef)
```

Comment:
Does this code have any copyright issues?  We need to be careful with incorporating code from other sources.